.github/workflows/test.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       matrix:
         python-version: [3.6, 3.7, 3.8]
-        jupyter-version: ["4.*", "5.6", "5.7", "5.*", "6.0", "6.*"]
+        jupyter-version: ["5.6", "5.7", "5.*", "6.0", "6.*"]
```

Comment:
Can you set the `requirements-package.txt` to `notebook>=5`

Reply:
should we do `notebook>=5.6` since that's what's being tested?

Reply:
I changed it to `notebook>=5.6`